b/id.go

```diff
@@ -0,0 +1,85 @@
+package xidb
+
+import (
+	"database/sql/driver"
+	"fmt"
+	"time"
+
+	"github.com/rs/xid"
+)
+
+// ID represents a unique request id
+type ID struct {
+	xid.ID
+}
+
+// New generates a globally unique ID
+func New() ID {
+	return ID{
+		ID: xid.New(),
+	}
+}
+
+// NewWithTime generates a globally unique ID with the passed in time
+func NewWithTime(t time.Time) ID {
+	return ID{
```

Comment:
Same comment, let the user do xidb.ID(xid.NewWithTime(t)) instead.

Reply:
That seems a lot of work for users, don't you think it's easier to just wrap it here?
Are you afraid of too much code duplication? Or potential change of interface that will require more work?